app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] This should be done with the `author ` attribute (foreign key in this table) to reference the `user `model
`belongs_to :author, class_name: 'User'`